examples/seq2seq/builtin_trainer/finetune.sh

```diff
@@ -1,10 +0,0 @@
-# the proper usage is documented in the README, you need to specify data_dir, output_dir and model_name_or_path
```

Comment:
why these *sh have been deleted? Just recently I worked hard at fixing tests that use those

Reply:
It's moved to the seq2seq folder.

Reply:
ah, ok, github said they were deleted.